base/ca/src/main/java/com/netscape/ca/CertificateAuthority.java

```diff
@@ -1844,10 +1845,15 @@ public SingleResponse processRequest(Request req) {
                 certStatus = new UnknownInfo();
             }
         } catch (EDBRecordNotFoundException e) {
-            // not found
-            certStatus = new GoodInfo(); // not issued not all
+            logger.debug(name + "cert record not found");
+            certStatus = new UnknownInfo(); // not issued by this CA
         } catch (EBaseException e) {
             // internal error
+            logger.debug(name + e.toString());
+            certStatus = new UnknownInfo();
+        } catch (Exception e) {
+            // safety net
+            logger.debug(name + e.toString());
             certStatus = new UnknownInfo();
         }
```

Comment:
Instead of adding another `catch` clause I think we can just replace the `EBaseException` with `Exception` in the previous clause.